### PR TITLE
Karim: Farmer Greetings UI Improvement

### LINKED ISSUE #5: FEATURE: Farmer greetings message UI improvement
### Have you updated the title?

- [X] I have updated the issue's title.

### As a...

user

### I can...

not be overwhelmed by the large font of the greeting text and also appreciate the artistic design of the message

### So that...

I can have an improved user experience

### Discussion

1. Having greeting messages in home page and common room page is a great idea in general, but currently the font size is too big and it's just a plain text with black background color.
2. Make the Greeting text slightly smaller, and maybe add movement, hover effect, animation, or gradient background color to make it more stand out.
3. The design is entirely up to the students.

### Acceptance Criteria

- [x] Home page greeting text should have smaller font size than before as well as more sophisticated CSS effects.
- [x] Play page greeting text should have smaller font size than before as well as more sophisticated CSS effects.

### Implementation Todos

Frontend:
- [ ] Home page greeting text should have smaller font size than before as well as more sophisticated CSS effects.
- [ ] Play page greeting text should have smaller font size than before as well as more sophisticated CSS effects.

Testing:
- [ ] Frontend: greeting text should have smaller font size and more sophisticated css effects, and students should write unit tests check those attributes.

### PR DESCRIPTION
**Pull request by KarimBenyassine1**
## Overview

In this PR, I changed the greetings message on the home and play page so that the fonts are smaller and there is a color gradient animation.

## Screenshots (Optional)
Be
Screen Shot 2023-11-24 at 8 00 43 PM
fore:
Screen Shot 2023-11-24 at 8 00 46 PM

After:
Screen Shot 2023-11-24 at 8 58 08 PM
Screen Shot 2023-11-24 at 8 57 58 PM

## Feedback Request (Optional)

Please give suggestions on the following:
- This method might not be the best way to do things. What do you think?
- The way this is implemented is the best I can think of. Is it acceptable?
- Where should these things go? I put them here, but it feels out of place.

## Future Possibilities (Optional)
- Cooler, eye-catching greeting animation

## Validation (Optional)

1. Download this branch.
2. Run the project.
3. Go to HomePage and PlayPage.
4. Check greetings header and whether it fits requirements.

## Tests

- [ ] Frontend Unit tests (`npm test`) pass
- [ ] Frontend Test coverage (`npm run coverage`) 100%
- [ ] Frontend Mutation tests (`npx stryker run`) 100% 

## Linked Issues

Closes #5

### CODE REVIEW

**Review by kaitntom — DISMISSED** [review]
LGTM! The animation and green color really stand out!

**Comment by pconrad** [comment]
The animation is nice!

I wonder about the contrast with the backgrounds though.  Looking at this in the context of the backgrounds (via the storybook), it's kind of hard to read, and I wonder whether this is an improvement?

home-page

Some possible solutions would be to add a background color to make the green text pop?  It might be opaque, or have  partial alpha transparency so that the background shows through a bit, but there's enough contrast that the words can be read easily.

**Review by pconrad — CHANGES_REQUESTED** [review]
The code looks fine, and the animation is nice.

However, in context, the text is hard to read against the background; see the animation I linked to in an earlier comment.

Please add links to the storybook in the description, and show screenshots that show the text against all of the backgrounds.   

Also: please make adjustments so that the text can be easily read against all of the backgrounds.

File: frontend/src/main/pages/HomePage.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.gradient-animation {
```
Comment (pconrad) [inline frontend/src/main/pages/HomePage.css:1]: There is a better place to put CSS, but we'll leave cleaning that up to another PR.

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**